sdk/monitor/Azure.ResourceManager.Monitor/src/autorest.md

```diff
@@ -193,4 +262,55 @@ directive:
         "readOnly": true,
         "type": "string"
       }
+  # remove unnecessary property for resources in action groups. Both of these are not used, and identity has an incorrect type.
+  - from: actionGroups_API.json
+    where: $.definitions.AzureResource.properties
+    transform: >
+      $["identity"] = undefined;
+      $["kind"] = undefined;
+  - from: scheduledQueryRule_API.json
+    where: $.definitions.Resource.properties
+    transform: >
+      $["kind"] = undefined;
+  - from: dataCollectionEndpoints_API.json
+    where: $.definitions.DataCollectionEndpointResource.properties
+    transform: >
+      $["kind"] = undefined;
+  - from: dataCollectionRules_API.json
+    where: $.definitions.DataCollectionRuleResource.properties
+    transform: >
+      $["kind"] = undefined;
+```
+
+## Tag: package-monitor-track2
```

Comment:
Consider to use the Readme.md file in the swagger spec repo directly

Reply:
They do not have a tag with all latest swagger yet